src/test/java/pages/RestaurantsPage.java

```diff
@@ -19,43 +19,47 @@ public class RestaurantsPage {
 
     public final HeaderGeneralPageComponent headerGlobal;
 
-    public RestaurantsPage(WebDriver driver)    {
+    public RestaurantsPage(WebDriver driver) {
         PageFactory.initElements(driver, this);
         this.driver = driver;
         headerGlobal = new HeaderGeneralPageComponent(driver);
     }
+
     @FindBy(xpath = "//main/div/div/div")
     private List<WebElement> restaurants;
 
-    private WebElement neededRestaurant(String restName)   {
+    private WebElement neededRestaurant(String restName) {
```

Comment:
I missed this method in previous review, can you please do same changes. (Overload and "magic variables" thing)

Reply:
Done